afm/logging.py

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+#
+import logging
+
+FORMATS = {}
+
+class FybrikFormatter(logging.Formatter):
+    def format(self, record):
+        log_fmt = FORMATS.get(record.levelno)
+        formatter = logging.Formatter(log_fmt, "%H:%M%p")
+        return formatter.format(record)
+
+logger = logging.getLogger('arrow-flight-module')
```

Comment:
I think we want a python logger that is independent of the arrow-flight-module.  We don't want every python component that needs to use logging to have to replicate all of this.  Right?